fonts/Roboto_Condensed/roboto-condensed.css

```diff
@@ -1,41 +1,7 @@
-@font-face {
```

Comment:
Used in the FAQ answers, maybe a weird design choice but it's in there.

Reply:
Nice catch!